app.py

```diff
@@ -1,13 +1,28 @@
+import pandas as pd 
+import numpy as np
 from flask import Flask 
```

Comment:
pls delete this import, happens two lines below

Reply:
lol what are you talking about ??